examples/nginx-access-logs-streaming/README.md

```diff
@@ -0,0 +1,640 @@
+*Stack 1: Filebeat + Kafka + Proton*
+
+![Vector + Proton](https://static.wixstatic.com/media/2d747e_1d01a2a6b0074a999bc8e3c091a26162~mv2.png/v1/fill/w_1036,h_458,al_c,q_90,enc_auto/2d747e_1d01a2a6b0074a999bc8e3c091a26162~mv2.png)
+*Stack 2: Vector + Proton*
+
+![Proton](https://static.wixstatic.com/media/2d747e_773438fadf4b4fb89f449294d928616d~mv2.png/v1/fill/w_1391,h_338,al_c,lg_1,q_90,enc_auto/2d747e_773438fadf4b4fb89f449294d928616d~mv2.png)
+*Stack 3: Proton Only*
+
+Out of the three stacks, the simplest to setup is the 3rd stack where only Timeplus Proton can be used for the log analysis and this is what will be used in this post.
+
+There's a caveat: the simpler architecture has an important limitation which is that real-time log analysis can add significant load to a server, if the server is under-resourced. 
+It can significantly degrade the performance of the server on which it is run and might not scale well for real-world use.
+
+To avoid the performance and scalability limitations of the selected stack, since the blog itself is already hosted on AWS, I'm going to perform the analysis using a separate EC2 instance. 
+
+I will install Timeplus Proton on a separate EC2 instance and make the `/var/log/nginx/` folder available over a readonly NFS share. In other words, the Ghost blog and Timeplus proton EC2 instances will be inside the same VPC subnet making it easy to securely share the access logs between the two instances over a readonly NFS share.
+ 
+This approach of using NFS (instead of [Amazon EFS](https://docs.aws.amazon.com/efs/latest/ug/whatisefs.html)) allows us to keep the costs of the solution down since there will be no egress fees while also being secure.
+
+## Sharing the Nginx Access Log over NFS
+The private IP addresses of the 2 EC2 instances I will be using for the NFS share are captured in the table below:
+| EC2 Server Instance | Private IP |
+|--------------|------------|
+| Ghost web blog  | `172.31.17.58` |
+| Timeplus Proton | `172.31.24.29` |
+
+
+### NFS Server Setup
+1. SSH into the server for the Ghost web blog.
+
+2. Install NFS server components:
+```bash
+sudo apt-get install nfs-kernel-server -y
+```
+
+3. Create the directory to be shared:
+```bash
+sudo mkdir -p /var/log/nginx
+```
+
+4. Configure the NFS export such that only the Timeplus Proton server can access it by whitelisting its (private) IP address:
+```bash
+echo "/var/log/nginx  172.31.24.29(ro,sync,no_subtree_check)" | sudo tee -a /etc/exports
+```
+
+5. Export the shared directory:
+```bash
+sudo exportfs -ra
+```
+
+6. Start the NFS Server on the Ghost blog:
+```bash
+sudo systemctl restart nfs-kernel-server
+```
+
+7. Confirm that the NFS export is active:
+```bash
+showmount -e
+Export list for ip-172-31-17-58:
+/var/log/nginx 172.31.24.29
+```
+
+8. Don't forget to add an inbound rule for NFS (on port `2049`) in the security group for the Ghost blog instance.
+
+
+### NFS Client Setup
```

Comment:
Maybe change this H3 to `NFS Client Setup and Timeplus Proton Setup`

Reply:
Fixed.